snapcraft_legacy/plugins/v2/_kernel_build.py

```diff
@@ -278,17 +278,20 @@ def _do_base_config_cmd(
         logger.info("Using ubuntu config flavour %s", config_flavour)
         conf_cmd = textwrap.dedent(
             """	echo "Assembling Ubuntu config..."
-	branch=$(cut -d'.' -f 2- < ${{KERNEL_SRC}}/debian/debian.env)
-	baseconfigdir=${{KERNEL_SRC}}/debian.${{branch}}/config
-	archconfigdir=${{KERNEL_SRC}}/debian.${{branch}}/config/${{DEB_ARCH}}
-	ubuntuconfig=${{baseconfigdir}}/config.common.ubuntu
-	archconfig=${{archconfigdir}}/config.common.${{DEB_ARCH}}
-	flavourconfig=${{archconfigdir}}/config.flavour.{config_flavour}
-	cat ${{ubuntuconfig}} ${{archconfig}} ${{flavourconfig}} \
-> {dest_dir}/.config 2>/dev/null || true
 	if [ -f ${{KERNEL_SRC}}/debian/rules ] && [ -x ${{KERNEL_SRC}}/debian/rules ]; then
+		# Generate Ubuntu kernel configs
+		pushd ${{KERNEL_SRC}}
+		fakeroot debian/rules clean genconfigs || true
+		popd
+
+		# Pick the right kernel .config for the targer arch and flavour
```

Comment:
`targer` should be `target`
This should allow the codespell lint test to pass.

Reply:
Fixed, thanks for noticing the typo!